meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
```

Comment:
That is an option as well theoretically - but at the time I tried it it was failing. Can't remember to many details but we can take a look and see how it behaves.